frontend/src/components/comp-top-table-bar/comp-top-table-bar.tsx

```diff
@@ -0,0 +1,106 @@
+import { useEffect, useRef, useState } from 'react';
+
+import { ButtonOutline } from '@components/common/button-outline/button-outline';
+import { TrashCanIcon } from '@components/common/icons/trash-can/trash-can';
+import { Spinner } from '@components/common/spinner/spinner';
+import {
+  useClearComparisonMutation,
+  useGetComparisonCarsQuery,
+} from '@store/queries/comparisons';
+import { clsx } from 'clsx';
+
+import { Comparison } from './components/comparison';
+import { NoActiveComparison } from './components/no-active-comparison';
+import styles from './components/styles.module.scss';
+
+export const CompTopTableBar = (): React.ReactElement => {
+  const { data, isLoading } = useGetComparisonCarsQuery();
+  const [clearTable] = useClearComparisonMutation();
+  const [isCleared, setIsCleared] = useState(false);
+  const ref = useRef<HTMLInputElement>();
+
+  const cars = data?.slice().sort((a, b) => a.position - b.position);
+  const ids = cars?.map((car) => car.id);
+  const initialData = {
+    cars,
+    carsPositions: ids,
+  };
+
+  const handleClearTable = async (): Promise<void> => {
+    await clearTable();
+  };
+
+  const handleClearBtnClick = (event: React.MouseEvent): void => {
+    event.stopPropagation();
+
+    setIsCleared(true);
+    handleClearTable();
+  };
+
+  useEffect(() => {
+    addEventListener(
+      'wheel',
+      () => {
+        const currentElement = ref.current;
+        if (currentElement) {
+          const onWheel = (e: WheelEvent): void => {
+            if (e.deltaY == 0) return;
+            e.preventDefault();
+            currentElement.scrollTo({
+              left: currentElement.scrollLeft + 5 * e.deltaY,
+              behavior: 'smooth',
+            });
+          };
+          currentElement.addEventListener('wheel', onWheel);
+          return (): void =>
+            currentElement.removeEventListener('wheel', onWheel);
+        }
+      },
+      { passive: false },
+    );
+  });
```

Comment:
Memory leak!

Add handled into a function inside the `useEffect` and remove the listener on unmount

Reply:
It looks like we don't need this functionality at all here. Because we gonna make all tables scroll in sync so scroll handling might be implemented outside of this component. 

It's not the part of this task I guess